src/routes/profile-route/profile-fests-route.jsx

```diff
@@ -75,91 +119,87 @@ export const ProfileFestsRoute = () => {
+                  new Date(fest.startDate.replace(/-/g, "/"))
+                );
+                const endDate = new Intl.DateTimeFormat("en-GB").format(
+                  new Date(fest.endDate.replace(/-/g, "/"))
                 );
-                const endDate = new Intl.DateTimeFormat("en-US").format(
-                  new Date(items.endDate)
+                const attendeeStatus = fest.attendees.some(
+                  (attendee) => attendee.username === currentUser.username
                 );
-                // const isClicked = items.some((click) => click.id === items.id);
+                const festDateInFuture = dayjs().isBefore(fest.endDate);
+                // const isClicked = fest.some((click) => click.id === fest.id);
                 return (
                   <div
-                    key={index}
+                    key={fest.id}
                     className="flex flex-row justify-between mb-5 border-b border-black"
                   >
                     <div className="mb-5">
                       <h3 className="font-arialBold mb-2.5">
-                        <Link to={`/fests/${items.id}`}>{items.name}</Link>
+                        <Link to={`/fests/${fest.id}`}>{fest.name}</Link>
                       </h3>
                       <p className="font-arialRegular mb-2.5">
                         {startDate + " - " + endDate}
                       </p>
-                      <div className="flex flex-row">
-                        {items.attendees.length <= 4
-                          ? items.attendees?.map((attendee) => {
-                              return (
-                                <Keyword
-                                  key={attendee.username}
-                                  className="h-31px space-x-2 space-y-2 bg-gray xs:space-x-2 xs:space-y-2 text-black text-center mr-2.5"
-                                  keyword={attendee.username}
-                                />
-                              );
-                            })
-                          : items.attendees.slice(0, 4).map((attendee) => {
-                              // needs to have {+ attendees.length - 5} to show how many attendees after 5
-                              return (
-                                <>
-                                  <div>
-                                    <Keyword
-                                      key={attendee.username}
-                                      className="h-31px space-x-2 space-y-2 bg-gray xs:space-x-2 xs:space-y-2 text-black text-center mr-2.5"
-                                      keyword={attendee.username}
-                                    />
-                                  </div>
-                                </>
-                              );
-                            })}
-                        {items.attendees.length > 4 ? (
-                          <Keyword
```

Comment:
Also getting errors that Keyword is undefined

Reply:
showing Keyword is imported on line 10 above. Do you show that as well?